decred/decred/dcr/dcrdata.py

```diff
@@ -998,14 +989,13 @@ def sendOutputs(self, outputs, keysource, utxosource, feeRate=None):
         changeScriptVersion = txscript.DefaultScriptVersion
         changeScriptSize = txscript.P2PKHPkScriptSize
 
-        relayFeePerKb = feeRate * 1e3 if feeRate else self.relayFee()
         for (i, txout) in enumerate(outputs):
-            checkOutput(txout, relayFeePerKb)
+            checkOutput(txout, relayFee)
 
```

Comment:
It was converted here. So I guess it did take atoms/b. Should I revert? Or try to make everything atoms/b? Either is fine with me, but some accepting atoms/byte and other atoms/kb is confusing.